Azure/azure-cli
# [Keyvault] `az keyvault key sign/verify`: Add new commands to support signing with keyvault key and verify the signature

**Related command**

`az keyvault key sign/verify`

**Description**

New commands:
- `az keyvault key sign` to support signing with keyvault key
- `az keyvault key verify`: to verify the signature

**Testing Guide**

- `az keyvault key sign -a RS256 --digest 12345678901234567890123456789012 -n testkey --vault-name ystestkv`
- `az keyvault key verify -a RS256 --digest 12345678901234567890123456789012 --signature  "gcjaksz9J736oP2bJ6k1vvhJL5aI6qEHx0HBvDz2jsfuRuaGmLP3VhozVUL7yuqHROHZLw72fnHb3NF1LXGowkniIJJaLb21Jj/DBXhz2rZLRFQr/KaghtIp4P0qWH3tOc0BLEjYNU4ZH6QFDhx0SthOdemEVjPeNXq3RqeZR8MMfx0B21f1SfocgiQJZaDg21D0akummh7EWMQAGVgEu26i78udyrntsJrd4LBhNOP/9m1Giq8YDph6ZnzLW1PF/sOo6OC4Who66uCvVw+0qdFLst19i+beW+EeWtXW/Mbm5oSHVSno4yFRwdUIXSTCrPhUdwMdB0o0GlrErNbEow==" -n testkey --vault-name ystestkv`



---

This checklist is used to make sure that common guidelines for a pull request are followed.
